scripts/ocsp-stapling_tls13multi.test

```diff
@@ -0,0 +1,534 @@
+ready_file2="$WORKSPACE"/wolf_ocsp_s2_readyF2$$
+ready_file3="$WORKSPACE"/wolf_ocsp_s2_readyF3$$
+ready_file4="$WORKSPACE"/wolf_ocsp_s2_readyF4$$
+ready_file5="$WORKSPACE"/wolf_ocsp_s2_readyF5$$
+printf '%s\n' "ready file 1:  $ready_file1"
+printf '%s\n' "ready file 2:  $ready_file2"
+printf '%s\n' "ready file 3:  $ready_file3"
+printf '%s\n' "ready file 4:  $ready_file4"
+printf '%s\n' "ready file 5:  $ready_file5"
+
+test_cnf="ocsp_s2.cnf"
+
+wait_for_readyFile(){
+
+    counter=0
+
+    while [ ! -s $1 -a "$counter" -lt 20 ]; do
+        if [[ -n "${2-}" ]]; then
+            if ! kill -0 $2 2>&-; then
+                echo "pid $2 for port ${3-} exited before creating ready file.  bailing..."
+                exit 1
+            fi
+        fi
+        echo -e "waiting for ready file..."
+        sleep 0.1
+        counter=$((counter+ 1))
+    done
+
+    if test -e $1; then
+        echo -e "found ready file, starting client..."
+    else
+        echo -e "NO ready file at $1 -- ending test..."
+        exit 1
+    fi
+
+}
+
+remove_single_rF(){
+    if test -e $1; then
+        printf '%s\n' "removing ready file: $1"
+        rm $1
+    fi
+}
+#create a configure file for cert generation with the port 0 solution
+create_new_cnf() {
+    echo "Random Ports Selected: $1 $2 $3 $4"
+
+    cat <<- EOF > $test_cnf
+    #
+    # openssl configuration file for OCSP certificates
+    #
+
+    # Extensions to add to a certificate request (intermediate1-ca)
+    [ v3_req1 ]
+    basicConstraints       = CA:false
+    subjectKeyIdentifier   = hash
+    authorityKeyIdentifier = keyid:always,issuer:always
+    keyUsage               = nonRepudiation, digitalSignature, keyEncipherment
+    authorityInfoAccess    = OCSP;URI:http://127.0.0.1:$1
+
+    # Extensions to add to a certificate request (intermediate2-ca)
+    [ v3_req2 ]
+    basicConstraints       = CA:false
+    subjectKeyIdentifier   = hash
+    authorityKeyIdentifier = keyid:always,issuer:always
+    keyUsage               = nonRepudiation, digitalSignature, keyEncipherment
+    authorityInfoAccess    = OCSP;URI:http://127.0.0.1:$2
+
+    # Extensions to add to a certificate request (intermediate3-ca)
+    [ v3_req3 ]
+    basicConstraints       = CA:false
+    subjectKeyIdentifier   = hash
+    authorityKeyIdentifier = keyid:always,issuer:always
+    keyUsage               = nonRepudiation, digitalSignature, keyEncipherment
+    authorityInfoAccess    = OCSP;URI:http://127.0.0.1:$3
+
+    # Extensions for a typical CA
+    [ v3_ca ]
+    basicConstraints       = CA:true
+    subjectKeyIdentifier   = hash
+    authorityKeyIdentifier = keyid:always,issuer:always
+    keyUsage               = keyCertSign, cRLSign
+    authorityInfoAccess    = OCSP;URI:http://127.0.0.1:$4
+
+    # OCSP extensions.
+    [ v3_ocsp ]
+    basicConstraints       = CA:false
+    subjectKeyIdentifier   = hash
+    authorityKeyIdentifier = keyid:always,issuer:always
+    extendedKeyUsage       = OCSPSigning
+EOF
```

Comment:
```suggestion
    cat <<- EOF > $test_cnf
	   #
	   # openssl configuration file for OCSP certificates
	   #

	   # Extensions to add to a certificate request (intermediate1-ca)
	   [ v3_req1 ]
	   basicConstraints       = CA:false
	   subjectKeyIdentifier   = hash
	   authorityKeyIdentifier = keyid:always,issuer:always
	   keyUsage               = nonRepudiation, digitalSignature, keyEncipherment
	   authorityInfoAccess    = OCSP;URI:http://127.0.0.1:$1

	   # Extensions to add to a certificate request (intermediate2-ca)
	   [ v3_req2 ]
	   basicConstraints       = CA:false
	   subjectKeyIdentifier   = hash
	   authorityKeyIdentifier = keyid:always,issuer:always
	   keyUsage               = nonRepudiation, digitalSignature, keyEncipherment
	   authorityInfoAccess    = OCSP;URI:http://127.0.0.1:$2

	   # Extensions to add to a certificate request (intermediate3-ca)
	   [ v3_req3 ]
	   basicConstraints       = CA:false
	   subjectKeyIdentifier   = hash
	   authorityKeyIdentifier = keyid:always,issuer:always
	   keyUsage               = nonRepudiation, digitalSignature, keyEncipherment
	   authorityInfoAccess    = OCSP;URI:http://127.0.0.1:$3

	   # Extensions for a typical CA
	   [ v3_ca ]
	   basicConstraints       = CA:true
	   subjectKeyIdentifier   = hash
	   authorityKeyIdentifier = keyid:always,issuer:always
	   keyUsage               = keyCertSign, cRLSign
	   authorityInfoAccess    = OCSP;URI:http://127.0.0.1:$4

	   # OCSP extensions.
	   [ v3_ocsp ]
	   basicConstraints       = CA:false
	   subjectKeyIdentifier   = hash
	   authorityKeyIdentifier = keyid:always,issuer:always
	   extendedKeyUsage       = OCSPSigning
	EOF
```

Reply:
Your suggested changes doesn't work on specific `bash` version regarding `EOF`. `EOF` needs to be always the beginning of the line.

```
The version doesn't work
$ bash --version
GNU bash, version 5.2.15(1)-release (x86_64-pc-linux-gnu)
$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Pengwin
Release:        12
Codename:       bookworm
```

```
This works.
$ bash --version
GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.4 LTS
Release:        22.04
Codename:       jammy
```

Reply:
Ok that makes sense. Thanks.